internal/plugins/ansible/v1/plugin.go

```diff
@@ -16,13 +16,13 @@ package ansible
 
 import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 
 	"github.com/operator-framework/operator-sdk/internal/plugins"
-	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 )
 
-const pluginName = "ansible" + plugins.DefaultNameQualifier
+const pluginName = "base.ansible" + plugins.DefaultNameQualifier
```

Comment:
we cannot change the name of the plugins in the PROJECT file. Then, would this change do it?

Reply:
I saw that it does not change the end-user config so 👍

Reply:
No, the project file will store the bundle name which is `"helm.sdk.operatorframework.io/v1"`. This is the main reason why the underlying plugin has been renamed to `base.*` so that they don't collide.